pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
+description = "Utilities for posting to LCLS Experimental ELog"
+dynamic = [ "version", "readme", "dependencies", "optional-dependencies", "optional-dependencies",]
+keywords = []
+name = "elog"
+requires-python = ">=3.9"
+
+[[project.authors]]
+name = "SLAC National Accelerator Laboratory"
+
+[project.scripts]
+LogBookPost = "elog.scripts.LogBookPost:main"
```

Comment:
I assume you tried this out and it works? 👍 if so

Reply:
I didn't even know this had a script built-in for that

Reply:
Yep, locally at least 👍